blazar_dashboard/content/leases/views.py

```diff
@@ -34,6 +34,10 @@
 from horizon import workflows
 from horizon.utils import memoized
 
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
nit: 
```suggestion
LOG = logging.getLogger(__name__)
```

And it might be more standard practice to use oslo_log instead of logging. Maybe check elsewhere in Horizon.